src/app/(sidebar)/smart-contracts/contract-explorer/page.tsx

```diff
@@ -124,6 +139,10 @@ export default function ContractExplorer() {
     !contractIdInput ||
     Boolean(contractIdInputError);
 
+  const isSacType = contractType
+    ? contractType === "contractExecutableStellarAsset"
+    : undefined;
+
```

Comment:
For Invoke Contract, currently, we're getting functions from the Wasm file. How should we get them for SAC? I tried to parse [the spec JSON from the repo](https://github.com/stellar/stellar-asset-contract-spec/blob/main/stellar-asset-spec.json), but it didn't like some characters. :pray:

Reply:
<img width="1166" height="123" alt="image" src="https://github.com/user-attachments/assets/c22830d8-b71f-44e6-8447-8b962a0a30ba" />

Reply:
When we buidl, we find 🐞

Reply:
@quietbits you could download the XDR then xdr2json it in the meantime? :shrug: though I'm not even sure that spec is up to date, given its age+archived status

Reply:
@Shaptic, is there another way we can get the XDR? The one in the repo gives us this: https://github.com/stellar/laboratory/pull/1510#discussion_r2216656850 :woman_shrugging:

Reply:
But that XDR seems valid - string comments in the spec (same way if you `grep`'d on a raw binary you'd see things like plaintext error strings in it). Idk why you're rendering it as text, though, as opposed to base64 or hex

Reply:
> 😕 that seems like a flaw of the spec itself. It looks like there should be a comma there to split the `}{`... Can we regenerate this with valid JSON, @leighmcculloch?

It's a stream of JSON objects. We can change the repo to generate a newline delimited stream, or an array instead. @quietbits Feel free to resurrect the repo and open a PR making it output an array.